usaspending_api/awards/models_matviews.py

```diff
@@ -558,87 +518,77 @@ class Meta:
 
 
 class TransactionMatView(models.Model):
-    # unique ids + cols used for unique id
-    detached_award_proc_unique = models.TextField()
     afa_generated_unique = models.TextField()
-    piid = models.TextField()
-    parent_award_id = models.TextField()
-    fain = models.TextField()
-    uri = models.TextField()
+    detached_award_proc_unique = models.TextField()
+
+    action_date = models.DateTimeField()
     agency_id = models.TextField()
-    referenced_idv_agency_iden = models.TextField()
+    assistance_type = models.TextField()
+    award_description = models.TextField()
     award_modification_amendme = models.TextField()
-    transaction_number = models.IntegerField()
-
-    # duns
-    awardee_or_recipient_uniqu = models.TextField()
     awardee_or_recipient_legal = models.TextField()
-
-    # recipient
+    awardee_or_recipient_uniqu = models.TextField()
+    awarding_office_code = models.TextField()
+    awarding_office_name = models.TextField()
+    awarding_sub_tier_agency_c = models.TextField()
+    base_and_all_options_value = models.DecimalField()
+    business_funds_indicator = models.TextField()
+    business_types = models.TextField()
+    cfda_number = models.TextField()
+    cfda_title = models.TextField()
+    contract_award_type = models.TextField()
+    contract_award_type_desc = models.TextField()
+    extent_compete_description = models.TextField()
+    extent_competed = models.TextField()
+    fain = models.TextField()
+    federal_action_obligation = models.DecimalField()
+    funding_office_code = models.TextField()
+    funding_office_name = models.TextField()
+    funding_sub_tier_agency_co = models.TextField()
+    idv_type = models.TextField()
+    idv_type_description = models.TextField()
+    last_modified_date = models.TextField()
     legal_entity_address_line1 = models.TextField()
     legal_entity_address_line2 = models.TextField()
     legal_entity_address_line3 = models.TextField()
-    legal_entity_foreign_provi = models.TextField()
+    legal_entity_city_name = models.TextField()
+    legal_entity_congressional = models.TextField()
     legal_entity_country_code = models.TextField()
     legal_entity_country_name = models.TextField()
-    legal_entity_state_code = models.TextField()
-    legal_entity_state_name = models.TextField()
     legal_entity_county_code = models.TextField()
     legal_entity_county_name = models.TextField()
-    legal_entity_city_name = models.TextField()
+    legal_entity_foreign_provi = models.TextField()
+    legal_entity_state_code = models.TextField()
+    legal_entity_state_name = models.TextField()
     legal_entity_zip5 = models.TextField()
-    legal_entity_congressional = models.TextField()
-
-    # place of performance
+    naics = models.TextField()
+    naics_description = models.TextField()
+    original_loan_subsidy_cost = models.DecimalField()
+    parent_award_id = models.TextField()
```

Comment:
Perhaps better to make this `parent_award_piid`

Reply:
updated!